docker-entrypoint.sh

```diff
@@ -22,7 +22,11 @@ fi
 if [ "$1" == "tiles" ]; then
     sudo -E -u postgres /var/lib/postgresql/src/generate_tiles.py
     if [ -n "${MAPNIK_TILE_S3_BUCKET}" ]; then
-        cd /var/lib/mod_tile/ && aws s3 sync . "s3://${MAPNIK_TILE_S3_BUCKET}/osm_tiles/" --size-only
+        if [ "${S3_FORCE_OVERWRITE}" == "1" ]; then
+            cd /var/lib/mod_tile/ && aws s3 sync . "s3://${MAPNIK_TILE_S3_BUCKET}/osm_tiles/"
+        else
+            cd /var/lib/mod_tile/ && aws s3 sync . "s3://${MAPNIK_TILE_S3_BUCKET}/osm_tiles/" --size-only
```

Comment:
Rather than a specific flag, what about something like `S3_SYNC_ARGS`?
```
 cd /var/lib/mod_tile/ && aws s3 sync . "s3://${MAPNIK_TILE_S3_BUCKET}/osm_tiles/" ${S3_SYNC_ARGS:- --size-only}
```

Reply:
So should we want to overwrite everything, we would need to set `S3_SYNC_ARGS` to an empty value?

1) I'm not sure if it's going to work, meaning `${S3_SYNC_ARGS:- --size-only}` will probably see no difference between empty value vs unset value and use `--size-only` in both cases. 
2) I don't think it's more convenient to whoever runs this script.

Reply:
Is there a way to drop the duplication at least?

Reply:
Done.